### PR TITLE
Fix ULong config boundary checking

### PR DESCRIPTION
**Pull request by BCathcart**
I noticed in #738 that we don't properly check ULong config boundaries and made the change there. I'm pulling out that particular commit into this PR since we don't know if we want to merge the configurable cluster blacklist TTL yet.

### CODE REVIEW

**Review by enjoy-binbin — APPROVED** [review]
this looks correct, it shouldn't be harmful to users, right?

**Comment by PingXie** [comment]
> this looks correct, it shouldn't be harmful to users, right?

they should be fine. they actually have a larger range to play with now with this change.

**Review by PingXie — APPROVED** [review]

**Comment by madolson** [comment]
> this looks correct, it shouldn't be harmful to users, right?

There is no harm, but I didn't see any real reason to do it. It's not like they are reasonable maximums anyways, nobody should increase the values that high. Why don't we just fix the boundary checking and leave the current limits alone?

File: src/config.c [hunk 1]
```diff
@@ -3170,9 +3171,9 @@ standardConfig static_configs[] = {
 #endif
 
     /* Unsigned Long configs */
-    createULongConfig("active-defrag-max-scan-fields", NULL, MODIFIABLE_CONFIG, 1, LONG_MAX, server.active_defrag_max_scan_fields, 1000, INTEGER_CONFIG, NULL, NULL), /* Default: keys with more than 1000 fields will be processed separately */
-    createULongConfig("slowlog-max-len", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.slowlog_max_len, 128, INTEGER_CONFIG, NULL, NULL),
-    createULongConfig("acllog-max-len", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.acllog_max_len, 128, INTEGER_CONFIG, NULL, NULL),
+    createULongConfig("active-defrag-max-scan-fields", NULL, MODIFIABLE_CONFIG, 1, ULONG_MAX, server.active_defrag_max_scan_fields, 1000, INTEGER_CONFIG, NULL, NULL), /* Default: keys with more than 1000 fields will be processed separately */
+    createULongConfig("slowlog-max-len", NULL, MODIFIABLE_CONFIG, 0, ULONG_MAX, server.slowlog_max_len, 128, INTEGER_CONFIG, NULL, NULL),
+    createULongConfig("acllog-max-len", NULL, MODIFIABLE_CONFIG, 0, ULONG_MAX, server.acllog_max_len, 128, INTEGER_CONFIG, NULL, NULL),
```
Comment (madolson) [inline src/config.c]:
```suggestion
    createULongConfig("active-defrag-max-scan-fields", NULL, MODIFIABLE_CONFIG, 1, LONG_MAX, server.active_defrag_max_scan_fields, 1000, INTEGER_CONFIG, NULL, NULL), /* Default: keys with more than 1000 fields will be processed separately */
    createULongConfig("slowlog-max-len", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.slowlog_max_len, 128, INTEGER_CONFIG, NULL, NULL),
    createULongConfig("acllog-max-len", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.acllog_max_len, 128, INTEGER_CONFIG, NULL, NULL),
```

I would rather just leave these alone and just fix the boundary checks.

Comment (BCathcart) [reply src/config.c]: You're right that it doesn't really matter, but I figured why not do it? I see generally most configs that don't have specific thresholds use the maximum value for whatever numeric type they are

Comment (madolson) [reply src/config.c]: > but I figured why not do it?

Makes it harder to find git blame, which isn't a big deal but I do prefer minimizing it.

Comment (BCathcart) [reply src/config.c]: I agree with that but the latest blame points to a commit reverting a formatting change so thought it was already a lost cause, but your call

**Review by madolson — APPROVED** [review]

**Pull request merged**